Source/Core/VideoBackends/Vulkan/CommandBufferManager.cpp

```diff
@@ -92,7 +93,10 @@ bool CommandBufferManager::CreateCommandBuffers()
       LOG_VULKAN_ERROR(res, "vkCreateSemaphore failed: ");
       return false;
     }
+  }
 
+  for (VkDescriptorPool& m_descriptor_pool : m_descriptor_pools)
```

Comment:
Please change to `for (VkDescriptorPool& descriptor_pool : m_descriptor_pools)`

The use of the `m_` notation is for member variables